lib/albacore/tasks/release.rb

```diff
@@ -34,7 +35,7 @@ class Release
       include ::Rake::DSL
       include ::Albacore::DSL
 
-      def initialize name = :release, opts = {}
+      def initialize name = :release, opts = {}, &block
```

Comment:
Unsure whether a block is best here, or a `with_package` parameter that accepts some kind of Proc